### PR TITLE
[ja] Localize Event-Driven Architecture into Japanese

### LINKED ISSUE #2855: [ja] Localize Event-Driven Architecture into Japanese
Localize https://glossary.cncf.io/event-driven-architecture/ into Japanese

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Event-Driven Architecture into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2855 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview site:

- https://deploy-preview-2987--cncfglossary.netlify.app/ja/event-driven-architecture/

**Review by naonishijima — APPROVED** [review]
LGTM

File: content/ja/event-driven-architecture.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+イベント駆動アーキテクチャは、イベントの作成、処理、および消費を促進するソフトウェアアーキテクチャです。
+イベントとは、アプリケーションの状態に対する任意の変更を指します。
+例えば、ライドシェアリングアプリで乗車を依頼することは、イベントを代表しています。
+このアーキテクチャは、イベントがそのソース(乗車を要求するアプリ)から望ましいレシーバー(近くの利用可能なドライバーのアプリ)へ適切にルーティングされる構造を作り出します。
+
+## 解決すべき問題はなんですか
+
+データがリアルタイムになるにつれて、イベントがキャプチャされ、イベントリクエストを処理する必要がある適切な[サービス](/ja/service/)へ正確にルーティングされる信頼性の高い方法を見つけることがますます困難になります。
```
Comment (kaitoii11) [inline content/ja/event-driven-architecture.md:15]:
```suggestion
より多くのデータがリアルタイムになるにつれて、イベントがキャプチャされ、イベントリクエストを処理する必要がある適切な[サービス](/ja/service/)へ正確にルーティングされる信頼性の高い方法を見つけることがますます困難になります。
```

File: content/ja/event-driven-architecture.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+イベントを処理する従来の方法は、メッセージが適切にルーティングされたか、あるいは実際に送信または受信されたかを保証する方法がないことがしばしばあります。
+アプリケーションがスケールするにつれて、イベントをオーケストレーションすることがより困難になります。
+
+## どのように役に立つのでしょうか
+
+イベント駆動アーキテクチャは、すべてのイベントのためのセントラルハブ(例えばKafka)を確立します。
+次に、イベントプロデューサー(ソース)とコンシューマー(レシーバー)を定義し、セントラルハブがイベントの流れを保証します。
+このアーキテクチャは、サービスが結合されずに残り、イベントがプロデューサーからコンシューマーに適切にルーティングされることを保証します。
```
Comment (kaitoii11) [inline content/ja/event-driven-architecture.md:23]:
```suggestion
このアーキテクチャは、サービス同士が疎結合のまま、イベントがプロデューサーからコンシューマーに適切にルーティングされることを保証します。
```

**Review by kaitoii11 — CHANGES_REQUESTED** [review]

**Comment by Okabe-Junya** [comment]
ありがとうございます〜！提案をcommitさせて頂きました

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**